src/pages/nerd-days.js

```diff
@@ -221,6 +221,7 @@ const NerdDaysPage = () => {
                 title="REGISTER FOR NERD DAYS | AMERICAS"
                 munchkinId="412-MZS-894"
                 publishableKey="pk_4d10daa544de6f993a9a9ce002ccd1c6"
+                thankYou="https://developer.newrelic.com/nerd-days-thank-you"
```

Comment:
```suggestion
                redirectLink="/nerd-days-thank-you"
```

This has to be a relative path, not a full external link. 
You can read more about this [here](https://www.gatsbyjs.com/docs/gatsby-link/#how-to-use-the-navigate-helper-function) 😄